### ISSUE TITLE
Delete docs in master branch?

### ISSUE THREAD

**Issue by stv0g** [SpectrumIM/spectrum2#344]
I got confused because there are currently two places where we have documentation for Spectrum2:

- `master` branch, `docs` directory (uses textile)
- `gh-pages` branch (uses Jekyll, which seems to be more up-to-date)

Should I create a PR, to delete the old docs?

**Comment by vitalyster** [comment]
1. Needs to review textile docs and move missing to gh-pages
2. Needs to integrate Doxygen docs somehow to other docs
3. We can move spectrum.im docs in separate repo inside org

**Comment by stv0g** [comment]
> We can move spectrum.im docs in separate repo inside org

Good idea. I moved the `gh-pages` into a separate repo: https://github.com/stv0g/spectrum-website
I made you a "collaborator" for this repo. This might allow you to move the repo into the SpectrumIM org.

> Needs to review textile docs and move missing to gh-pages

Done. I made some changes to the repo above.

> Needs to integrate Doxygen docs somehow to other docs:

WIP: https://github.com/stv0g/spectrum2/tree/deploy-doxygen

I left the `Doxyfile` in the spectrum2 repo and added a `deploy` section to `travis-ci.yml` which uploads the generated Doxygen HTML output to GitHub Pages.

WIP. I left the
